### ISSUE TITLE
Fix code scanning alert - Images should have alternate text (alt text)

### ISSUE THREAD

**Issue by MasahiroSakoda** [MasahiroSakoda/dotfiles#279]
Tracking issue for:
- [x] https://github.com/MasahiroSakoda/dotfiles/security/code-scanning/370
- [x] https://github.com/MasahiroSakoda/dotfiles/security/code-scanning/371
- [x] https://github.com/MasahiroSakoda/dotfiles/security/code-scanning/372
